src/ora/test/regression_tests.tcl

```diff
@@ -0,0 +1,5 @@
+record_tests {
+  set_bothost1
+  set_consent1
+  askbot1
+}
```

Comment:
all the testing needs to be moved to the CTest as this was removed.

Reply:
I've moved the tests, but still have to figure out a way to start the python server (to mock API responses for tests).  Previously, I started the Python server within the regression bash script itself. Could you suggest ways to handle this directly in CTest?

Reply:
I've rewritten the tests to use CTest and have implemented the requested changes. Please let me know if any further modifications are needed.